tests/DCPS/DPFactoryQos/rtps_disc.ini

```diff
@@ -0,0 +1,13 @@
+[common]
+DCPSGlobalTransportConfig=$file
+
+[domain/411]
```

Comment:
Domain IDs above 231 are not supported by RTPS Discovery (see docs/design/RTPS and devguide).